code/modules/food_and_drinks/recipes/processor_recipes.dm

```diff
@@ -9,8 +9,8 @@
 	var/time = 40
 	/// The machine required to do this recipe
 	var/required_machine = /obj/machinery/processor
-	/// The number of products this recipe creates.
-	var/multiplier = 1 //This multiplies the number of products produced per object processed.
+	/// Multiplied additional food made when processed
+	var/food_multiplier = 0
```

Comment:
0 * anything is 0, you might wanna change this to 1

Reply:
i forgot to revert this when i reverted everything else to make it multiply, thanks for catching

Reply:
also i wish you made it a suggestion so I could make you co-author for credit